.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,14 +14,12 @@ If adding a new definition:
 - [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
 - [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
 - [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
-- [ ] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
```

Comment:
I think that it's good to still keep a reminder in the template.  Instead of removing it, it could be trimmed to be a quick reference to the README file section.

Reply:
Done :heavy_check_mark: